test/integration/mod/ican.mod

```diff
@@ -0,0 +1,56 @@
+: simple first-order model of calcium dynamics
+
+        NEURON {

```

Comment:
I would do minimal mod file. (remove all other part of mod file which are not required to test  this ISPC keyword issue)